acceptancetests/jujupy/client.py

```diff
@@ -50,6 +47,35 @@
     get_juju_home,
     get_selected_environment,
     )
+from jujupy.exceptions import (
```

Comment:
While import * isn't always the right solution perhaps it makes sense if we're importing everything by name?